platform/core.multitabs/src/org/netbeans/core/multitabs/impl/DocumentSwitcherTable.java

```diff
@@ -174,7 +177,27 @@ public String getToolTipText( MouseEvent event ) {
         }
         return null;
     }
-
+    
+    boolean closeSelectedDocumentList() {
+        List<TabData> tabs = controller.getTabModel().getTabs();
+        Item item = ( Item ) getModel().getValueAt( getSelectedRow(), getSelectedColumn());
+        ProjectProxy project = item.getProject();
+        ProjectSupport projectSupport = ProjectSupport.getDefault();
+        int numOfOtherTabs = 0;
+        for ( int i = tabs.size(); i-- > 0; ) {  //reverse iteration to close tabs from the end so tabIndex does not change
+            TabData projectTab = tabs.get( i );
+            ProjectProxy projectForTab = projectSupport.getProjectForTab( projectTab );
+            if (( project == null && projectForTab == null ) || ( projectForTab != null && projectForTab.equals( project ))) {
```

Comment:
Why would the tab be closed in the project == null && projectForTab == null case?

Reply:
There is a case, when no project is associated when file and variables projectForTab and project are null. Such as here for example: 
![image](https://user-images.githubusercontent.com/94047859/200135894-ab5d695b-9fd7-4203-a3f0-65d293dc3090.png)

Reply:
Makes sense. Thanks!